packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -75,6 +75,10 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     this.outcomeAddress = null;
   }
 
+  get(torrentId: PaidStreamingTorrent | string | Buffer): PaidStreamingTorrent | void {
+    return super.get(torrentId) as PaidStreamingTorrent;
```

Comment:
Is `get` overridden to enable this type assertion?

Reply:
Yes